static/app/components/stream/group.tsx

```diff
@@ -596,55 +685,70 @@ function StreamGroup({
         </FirstSeenWrapper>
       )}
 
-      {withChart && !displayReprocessingLayout ? (
+      {withChart && !displayReprocessingLayout && (
         <ChartWrapper breakpoint={COLUMN_BREAKPOINTS.TREND}>
-          {issueTypeConfig.stats.enabled ? (
+          {issueTypeConfig.stats.enabled && defined(groupStats) ? (
             <GroupStatusChart
               hideZeros
-              loading={!defined(groupStats)}
               stats={groupStats}
               secondaryStats={groupSecondaryStats}
               showSecondaryPoints={showSecondaryPoints}
               groupStatus={getBadgeProperties(group.status, group.substatus)?.status}
               showMarkLine
             />
+          ) : issueTypeConfig.stats.enabled ? (
+            <Placeholder height="36px" />
           ) : null}
         </ChartWrapper>
-      ) : null}
+      )}
       {displayReprocessingLayout ? (
         renderReprocessingColumns()
       ) : (
         <Fragment>
           {showLastTriggered && (
             <LastTriggeredWrapper>{lastTriggered}</LastTriggeredWrapper>
           )}
-          {withColumns.includes('event') ? (
+          {withColumns.includes('event') && (
             <NarrowEventsOrUsersCountsWrapper breakpoint={COLUMN_BREAKPOINTS.EVENTS}>
-              {issueTypeConfig.stats.enabled ? groupCount : null}
+              {issueTypeConfig.stats.enabled && defined(primaryCount) ? (
+                groupCount
+              ) : issueTypeConfig.stats.enabled ? (
+                <Placeholder height="18px" width="40px" />
+              ) : null}
             </NarrowEventsOrUsersCountsWrapper>
-          ) : null}
-          {withColumns.includes('users') ? (
+          )}
+          {withColumns.includes('users') && (
             <NarrowEventsOrUsersCountsWrapper breakpoint={COLUMN_BREAKPOINTS.USERS}>
-              {issueTypeConfig.stats.enabled ? groupUsersCount : null}
+              {issueTypeConfig.stats.enabled && defined(primaryUserCount) ? (
+                groupUsersCount
+              ) : issueTypeConfig.stats.enabled ? (
+                <Placeholder height="18px" width="40px" />
+              ) : null}
             </NarrowEventsOrUsersCountsWrapper>
-          ) : null}
+          )}
           {withColumns.includes('priority') ? (
             <PriorityWrapper breakpoint={COLUMN_BREAKPOINTS.PRIORITY}>
               {group.priority ? (
                 <GroupPriority group={group} onChange={onPriorityChange} />
-              ) : null}
+              ) : (
+                <Placeholder height="24px" />
```

Comment:
`group.priority` doesn't get loaded in async, so better to just render null in this case (though I'm not sure if there is a case where `priority` isn't available)